src/commands/index.ts

```diff
@@ -26,9 +26,10 @@ export function load(client: any): any {
   client.appdistribution.testers.add = loadCommand("appdistribution-testers-add");
   client.appdistribution.testers.delete = loadCommand("appdistribution-testers-remove");
   client.appdistribution.group = {};
```

Comment:
sorry, one more: `client.appdistribution.groups` (i.e. plural)?

Reply:
See line 32 - we copy it over from client.appdistribution.group instead of calling loadCommand 2x, so that we don't register the commands 2x

Reply:
What I meant was swapping `group` for `groups` on lines 29-31 and reversing the assignment on line 32. But it doesn't really matter. We're deep in supernit territory. Sorry.